training/model_evaluation/environment/context/Dockerfile

```diff
@@ -16,3 +16,7 @@ ENV PATH=$CONDA_PREFIX/bin:$PATH
 
 # This is needed for mpi to locate libpython
 ENV LD_LIBRARY_PATH $CONDA_PREFIX/lib:$LD_LIBRARY_PATH
+
+RUN pip install azureml-acft-common-components["image"]=={{latest-pypi-version}}
```

Comment:
Hi @MadhuM02 I think you'll have to remove the {{latest-pypi-version}} from dockerfile. 

It is not getting rendered in Dockerfile it seems. We'll just have to keep the latest version. Apologies for the msinformation.

Reply:
Actual latest version*

Reply:
No worries Ankesh, have changed it to actual version